lib/ton/address.ex

```diff
@@ -57,13 +48,11 @@ defmodule Ton.Address do
 
   @spec raw_address_to_friendly_address(binary(), Keyword.t()) :: binary()
   def raw_address_to_friendly_address(raw_address, opts \\ []) do
```

Comment:
can we  rename this function to `raw_address_to_friendly_address!` because it could fail ?